apps/meteor/client/views/marketplace/AppMenu.js

```diff
@@ -344,7 +344,7 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 								{t('Enable')}
 							</>
 						),
-						disabled: !appCountQuery?.data?.hasUnlimitedApps && appCountQuery?.data?.enabled >= appCountQuery?.data?.limit,
+						disabled: !app.migrated && !appCountQuery?.data?.hasUnlimitedApps && appCountQuery?.data?.enabled >= appCountQuery?.data?.limit,
```

Comment:
question, is this just a client validation? isn't there anything preventing them from being enabled on server side as well?

Reply:
There is, yes